src/index.js

```diff
@@ -126,34 +141,53 @@ const runAudit = async ({ path, url, thresholds }) => {
       return {
         summary,
         shortSummary,
-        error: errors.length > 0 ? new Error(`\n${errors.join('\n')}`) : false,
+        errors,
```

Comment:
Don't format the message here so we can create both a short message and a detailed message later on.